conduit/plugins/importers/filereader/filereader_test.go

```diff
@@ -81,7 +81,7 @@ func initializeImporter(t *testing.T, numRounds int) (importer importers.Importe
 	tempdir = t.TempDir()
 	genesisExpected := initializeTestData(t, tempdir, numRounds)
 	importer = New()
-	defer importer.Config()
+	// defer importer.Config()
```

Comment:
I'm not sure what the purpose of this statement is. I don't see any clean up happening in the deferred function.